### PR TITLE
profiles: adding optimization -O2 to profile

### PR DESCRIPTION
**Pull request by staiyeba**

### CODE REVIEW

**Comment by mnordsletten** [comment]
Great stuff. I want a review from both @fwsGonzo and @KristianJerpetjon on this one before we merge 👍

**Review by fwsGonzo — APPROVED** [review]

**Review by KristianJerpetjon — APPROVED** [review]
LGTM

**Pull request merged**
